app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -23,12 +23,18 @@ $gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
 }
 
 .gem-c-button--inline {
-  width: auto;
+  display: block;
+  width: 100%;
   margin-bottom: govuk-spacing(1);
-  margin-right: govuk-spacing(2);
 
-  @include govuk-media-query($from: tablet) {
-    margin-bottom: 0;
+  @include govuk-media-query($from: mobile) {
+    display: inline-block;
+    box-sizing: border-box;
```

Comment:
Is there a mixin we should be using for this? Or is it okay?

Reply:
🤦‍♀ I always forget about this mixin. Thanks!